packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -0,0 +1,336 @@
+import 'dart:convert';
+
+import 'package:flutter_tools/src/base/process.dart';
+import 'package:flutter_tools/src/base/terminal.dart';
+import 'package:intl/intl.dart';
+import 'package:meta/meta.dart';
+import 'package:vm_snapshot_analysis/treemap.dart';
+
+import '../base/file_system.dart';
+import '../globals.dart' as globals;
+
+// This file can be in devtools_shared
+class SizeAnalyzer {
+  static const String kAotSizeJson = 'aot-size.json';
```

Comment:
prefixing constants with k is an old convention that I hope Flutter tools is moving away from.
consider calling this constant aotSizeFileName as it is a file name not the json content.